android/jni/app/organicmaps/Framework.cpp

```diff
@@ -1103,11 +1103,14 @@ Java_app_organicmaps_Framework_nativeGetMovableFilesExts(JNIEnv * env, jclass)
   return jni::ToJavaStringArray(env, exts);
 }
 
+std::array<std::string, 4> const kBookmarkExtensions = {
+    std::string{kKmzExtension}, std::string{kKmlExtension}, std::string{kKmbExtension}, std::string{kGpxExtension}
+};
```

Comment:
- so what with string_view then?
- should be static in function scope then?

Reply:
Done.

JNI's toString unfortunately always takes a null-terminated string. There is no benefit in passing there string_view.